flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/TaskManagerServicesConfiguration.java

```diff
@@ -155,6 +155,10 @@ public int getPageSize() {
 		return pageSize;
 	}
 
+	public TaskExecutorResourceSpec getTaskExecutorResourceSpec() {
```

Comment:
Previously we do not have any cases that needs to access this field.
The public getter is introduced because we need to use it in `TaskManagerServices#fromConfiguration` in this commit.